code/modules/fallout/reagents/medicines.dm

```diff
@@ -373,20 +373,20 @@
 	color = "#6D6374"
 	metabolization_rate = 0.25 * REAGENTS_METABOLISM
 	overdose_threshold = 16
-	addiction_threshold = 6
+	addiction_threshold = 5
```

Comment:
A jab of med-x gives you 5 units and if 5 units is the addiction threshold, it means that whenever you've even jabbed yourself once.. you're addicted.

Reply:
Winners don't do drugs. It'd be better if we had a daytripper perk that made all addiction thresholds higher for people like raiders but generally speaking Med-X is one of the most addictive chems.

This is the problem with having an addiction threshold instead of just a random addiction rate.

Reply:
> This is the problem with having an addiction threshold instead of just a random addiction rate.

Then code a random addiction rate. 

> It'd be better if we had a daytripper perk that made all addiction thresholds higher for people like raiders

We can do something like that, or try to at least.

Reply:
should be resolved unless i'm a moron (i probably am)